src/pages/experiments/[experimentId]/plots-and-tables/marker-heatmap/index.jsx

```diff
@@ -313,6 +331,72 @@ const MarkerHeatmap = ({ experimentId }) => {
     return (<Skeleton />);
   }
 
+  const getCellOptions = (type) => {
+    const filteredOptions = hierarchy.filter((element) => (
```

Comment:
This looks a lot like a selector that stefan created, is there any chance we could reuse it?

Reply:
definitely!